packages/grafana-ui/src/components/FormattedValueDisplay/FormattedValueDisplay.mdx

```diff
@@ -0,0 +1,14 @@
+import { Props } from '@storybook/addon-docs/blocks';
+import { FormattedValueDisplay } from './FormattedValueDisplay';
+
+# FormattedValueDisplay
+
+Used to display a value, which also supports prefix and suffix.
```

Comment:
@ryantxu I couldn't figure proper usage and description here, maybe you have some opinions?